test/api/routes/user.test.ts

```diff
@@ -0,0 +1,110 @@
+import request from "supertest";
+import express, { Application, Router } from "express";
+import userRoutes from "../../../src/api/routes/user"
+import UserService from "../../../src/services/UserService";
+import UserRepositoryService from "../../../src/services/UserRepositoryService";
+import {Connection, createConnection} from "typeorm";
+
+jest.mock(
+    "../../../src/api/middlewares/validateToken",
+    () => jest.fn(() => (req, res, next) => next()),
+);
+jest.mock(
+    "../../../src/api/middlewares/logRequest",
+    () => jest.fn(() => (req, res, next) => next()),
+);
+
+jest.setTimeout(1000 * 60 * 60);
+
+describe("Testing user routes", () => {
+    let app: Application;
+    let connection: Connection;
+    let createdUserId: number;
+
+    const userDTO = {
+        "login": "Test user 111",
+        "password": "password",
+        "age": 25
+    }
+
+    beforeAll(async () => {
+        app = express();
+        app.use(express.json());
```

Comment:
for unit tests it's not needed to start the app. Your tests should be isolated and any requests to db or third party services must be mocked